src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/DefaultComesLast/context/defaultContext.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Test.java b/Test.java
+index be9c189..061cde2 100644
+--- a/Test.java
++++ b/Test.java
+@@ -9,6 +9,8 @@ public class Test {
+                 break;
+             default:  // violation context
```

Comment:
the default should be after all the cases in a switch statement, and `case 3` is not `default`'s child nodes.